Makefile

```diff
@@ -78,12 +78,12 @@ LIBS=be localestub translation bnetapi $(STDCPPLIBS)
 #	be recursive, so include all of the paths where libraries can
 #	be found.  Directories where source files are found are
 #	automatically included.
-LIBPATHS=
+LIBPATHS=/boot/system/develop/lib/ /boot/system/develop/lib/x86/
```

Comment:
Actually it's even confusing to add both gcc2 and gcc4 header paths, the makefile engine should take care of it already.